lib/takeaway.rb

```diff
@@ -0,0 +1,42 @@
+require 'twilio-ruby'
+
+class Takeaway
+
+  attr_reader :order, :menu
+
+  def initialize
+    @order = Order.new
+    @menu = Menu.new
+  end
+
+  def select_dish(dish)
+    @order.dishes.push(dish)
+  end
+
```

Comment:
Is the user able to edit their order (e.g. delete dishes) while they're compiling it? Can they view it before they submit it?